test/test.go

```diff
@@ -506,12 +506,7 @@ func (h *Helper) Path(name string) string {
 		joined = filepath.Join(h.tempdir, name)
 	}
 
-	// Ensure it's the absolute, symlink-less path we're returning
-	abs, err := filepath.EvalSymlinks(joined)
```

Comment:
In the test `TestIntegration/init/manifest-exists` it doesnt like doing the SymLinks checkas the  `lock` file doesnt exist in final and so the code I removes gives me this error:

```
lstat /private/var/folders/q9/jmtltdhs5hd8574dpdqgn40c0000gn/T/gotest220558389/src/github.com/golang/notexist/lock.json: no such file or directory
```

However, adding that file in  `/github.com/golang/dep/cmd/dep/testdata/harness_tests/init/manifest-exists/final` seems abit counter intuative as when the tests ends it shouldnt be there as we are testing that if a manifest already exists for a reason we error before generating a `lock` file.

If I created a new method we would have to change the line https://github.com/golang/dep/blob/master/cmd/dep/integration_test.go#L60 so that it got to call the new method that handled the files not existing.

We could however just return an empty string if we got an error in from `EvalSymlinks`.